components/compositing/constellation.rs

```diff
@@ -638,17 +638,11 @@ impl<LTF: LayoutTaskFactory, STF: ScriptTaskFactory> Constellation<LTF, STF> {
 
         self.close_pipeline(pipeline_id, ExitPipelineMode::Force);
 
-        loop {
```

Comment:
Hmm, maybe not. Looks like `pending_frames` is changed by `close_pipeline`; not sure how this works.